amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-studio.adoc

```diff
@@ -43,23 +43,29 @@ Adding a connector to a Mule project in Studio does not make that connector avai
 A source initiates a flow when a specified condition is met.
 You can configure one of these sources to use with Amazon S3:
 
-* HTTP Listener
+* *On Deleted Object*
 +
-Initiates a flow each time it receives a request on the configured host and port
+Initiates access to your app when an Amazon S3 object is deleted
```

Comment:
From the template (applies to all sources): "Initiates a flow when..."

Reply:
That seems to be the description for HTTP Listener. There are separate ones for the Amazon ones for On Deleted Object and On New Object.

Reply:
The "Initiates a flow when..." wording applies to all sources. Each source kicks off a flow, so it is good to be consistent with the first part of the description. I can add something to the template about this.